guides/source/getting_started.md

```diff
@@ -92,6 +92,9 @@ The version shown should be Rails 8.1.0 or higher.
 
 ### Creating Your First Rails App
 
+Note: When generating a new Rails application we use `rails`.
+But we should run commands  with `bin/rails` when running them within an application.
```

Comment:
I've expanded the explanation a little, to hopefully make it a bit clearer.
Also, using uppercase `NOTE` makes sure the note gets [higlighted ](https://edgeguides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#notes-tips-and-warnings):
```suggestion
NOTE: When generating a new Rails application we run the `rails` command, which
uses the latest installed version of Rails. When we run commands inside an
application directory, we should use `bin/rails`. This makes sure the application's
version of Rails is used.
```